examples/h2_server/source/app.d

```diff
@@ -0,0 +1,27 @@
+import vibe.appmain;
+import vibe.http.server;
+import vibe.stream.tls;
+import vibe.stream.botan;
+import vibe.core.log;
+import vibe.core.core;
+import libasync.threads;
+import std.datetime;
+
+void handleRequest(scope HTTPServerRequest req, scope HTTPServerResponse res)
+{
+	res.writeBody(cast(ubyte[])"Hello, World!", "text/plain");
```

Comment:
I actually have the `write` methods implemented as a `const(ubyte)[]` in botan, which makes a lot more sense. I haven't seen any implementations of the `write` method mutating the data, in fact they're usually copied to a buffer before being mutated, for this reason this cast doesn't produce any bugs.